src/models/Person.ts

```diff
@@ -14,6 +14,10 @@ class Person implements Model {
   picture?: File;
   is_active?: boolean;
   external_source_id?: string;
+  seatName?: string;
+  seatPicture?: File;
+  seatPictureRef?: string;
+  seatElectoralArea?: string;
```

Comment:
I don't think it makes sense to have these fields here because the person collection in the database doesn't have these fields. If you need these fields somewhere else in the code to exist on a person, you can extend the Person class to have those fields.

Reply:
You are correct!